src/natcap/invest/ndr/ndr.py

```diff
@@ -576,6 +593,10 @@ def execute(args):
         args['biophysical_table_path'],
         **MODEL_SPEC['args']['biophysical_table_path'])
 
+    # Ensure that if user doesn't explicitly assign a value, runoff_proxy_av = None
```

Comment:
```suggestion
    # Ensure that if user doesn't explicitly assign a value,
    # runoff_proxy_av = None
```

I think this was a few characters over 80 for line length.